template.tpl

```diff
@@ -104,642 +104,582 @@ ___TEMPLATE_PARAMETERS___
+                "value": "category"
+              },
               {
-                "selectItems": [
-                  {
-                    "displayValue": "category (required)",
-                    "value": "category"
-                  },
-                  {
-                    "displayValue": "variable (required)",
-                    "value": "variable"
-                  },
-                  {
-                    "displayValue": "timing (required)",
-                    "value": "timing"
-                  },
-                  {
-                    "displayValue": "label",
-                    "value": "label"
-                  }
-                ],
-                "defaultValue": "category",
-                "displayName": "Parameter Name",
-                "name": "name",
-                "isUnique": true,
-                "type": "SELECT"
+                "displayValue": "variable (required)",
+                "value": "variable"
               },
               {
-                "defaultValue": "",
-                "displayName": "Parameter Value",
-                "name": "value",
-                "type": "TEXT"
+                "displayValue": "timing (required)",
+                "value": "timing"
+              },
+              {
+                "displayValue": "label",
+                "value": "label"
               }
             ],
-            "type": "SIMPLE_TABLE",
-            "newRowButtonText": "Add Parameter"
-          }
-        ]
-      },
-      {
-        "help": "\u003ca href\u003d\"https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#error-tracking\"\u003eRead more\u003c/a\u003e about tracking error events.",
-        "enablingConditions": [
+            "defaultValue": "category",
+            "displayName": "Parameter Name",
+            "name": "name",
+            "isUnique": true,
+            "type": "SELECT"
+          },
           {
-            "paramName": "eventType",
-            "type": "EQUALS",
-            "paramValue": "trackError"
+            "defaultValue": "",
+            "displayName": "Parameter Value",
+            "name": "value",
+            "type": "TEXT"
           }
         ],
-        "displayName": "Error Tracking",
-        "name": "errorConfig",
-        "groupStyle": "NO_ZIPPY",
-        "type": "GROUP",
-        "subParams": [
+        "type": "SIMPLE_TABLE",
+        "newRowButtonText": "Add Parameter"
+      }
+    ]
+  },
+  {
+    "help": "\u003ca href\u003d\"https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#error-tracking\"\u003eRead more\u003c/a\u003e about tracking error events.",
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "trackError"
+      }
```

Comment:
This should probably be disabled when `paramsFromVariable` is set.

Reply:
I accidentally fixed this too when doing https://github.com/snowplow/snowplow-gtm-tag-template-v4/pull/4/commits/a7da3edcbdf854ee87b99fbc081e7f6e9cc11ff9 😄